scripts/index.js

```diff
@@ -0,0 +1,128 @@
+const state = {
+    tempCount: 70
+};
+
+
+const updateTemp = tempValue => {
+    const tempCountContainer = document.getElementById("tempValue");
+    tempCountContainer.textContent = tempValue;
+}
+// Wave 2 - Increase Temp Behavior
+const addNum = () => {
+    state.tempCount += 1;
+    updateTemp(tempValue);
```

Comment:
The `tempValue` and the `state.tempCount` are not connected here. Either approach will work  (using state with `state.tempCount` or passing an argument like `tempValue`), but you need to use one approach or the other.